test/integration/generated/healthcareapis/src/healthCareApisClient.ts

```diff
@@ -64,9 +64,6 @@ export class HealthCareApisClient extends coreClient.ServiceClient {
         ? `${options.userAgentOptions.userAgentPrefix} ${packageDetails}`
         : `${packageDetails}`;
 
-    if (!options.credentialScopes) {
-      options.credentialScopes = ["https://management.azure.com/.default"];
-    }
```

Comment:
There's an add-credential: false in the swagger config, I guess removing it is the correct behavior.